Common/Setup.py

```diff
@@ -74,9 +74,11 @@ def apply_customisations(config_dict, customisations):
         key_entries = substrings[0].split('.')
         cfg_entry = config_dict
         for key in key_entries[:-1]:
-            cfg_entry = cfg_entry[key]
-        entry_type = type(cfg_entry[key_entries[-1]])
-        cfg_entry[key_entries[-1]] = entry_type(value)
+            if key in config_dict.keys():
+                cfg_entry = cfg_entry[key]
+        if key_entries[-1] in config_dict.keys():
+            entry_type = type(cfg_entry[key_entries[-1]])
+            cfg_entry[key_entries[-1]] = entry_type(value)
```

Comment:
Undo modifications: if customization is not applied, the code shouldn't silently ignore it.